app/controllers/occupation_standards_controller.rb

```diff
@@ -1,6 +1,6 @@
 class OccupationStandardsController < ApplicationController
   def index
-    @occupation_standards_search = OccupationStandardQuery::Container.new(search_term: search_term_params)
+    @occupation_standards_search = OccupationStandardQuery::Container.new(q: params[:q])
```

Comment:
Why not keep using `search_term_params`?

Reply:
Oh, I think I see the confusion. Could we do:

```
search_term: search_term_params
```

like before, but have `search_term_params = { q: q }` ?